sky/provision/gcp/constants.py

```diff
@@ -180,6 +181,13 @@
     'resourcemanager.projects.get',
     'resourcemanager.projects.getIamPolicy',
 ]
+# If specifying custom VPC, permissions to modify network are not necessary
+# unless opening ports (e.g., via `resources.ports`).
+if skypilot_config.get_nested(("gcp", "vpc_name"), ""):
+    remove = ("compute.firewalls.create", "compute.firewalls.delete")
+    VM_MINIMAL_PERMISSIONS = [
+        p for p in VM_MINIMAL_PERMISSIONS if p not in remove
+    ]
```

Comment:
We are using single quotes in our codebase : )
```suggestion
if skypilot_config.get_nested(('gcp', 'vpc_name'), ''):
    remove = ('compute.firewalls.create', 'compute.firewalls.delete')
    VM_MINIMAL_PERMISSIONS = [
        p for p in VM_MINIMAL_PERMISSIONS if p not in remove
    ]
```

Reply:
Ah, sorry I thought `format.sh` would have done the quotes (currently using `black` which makes everything double quote) 👍